pkg/maps/lbmap/lbmap.go

```diff
@@ -398,6 +399,13 @@ func (*LBBPFMap) UpdateSourceRanges(revNATID uint16, prevSourceRanges []*cidr.CI
 
 	srcRangeMap := map[string]*cidr.CIDR{}
 	for _, cidr := range sourceRanges {
+		// k8s api server does not catch the IP family mismatch, so we need to catch it here
+		if ip.IsIPv6(cidr.IP) && !ipv6 {
```

Comment:
I think we need to cover a case when a service frontend is IPv6, but the source range entry is IPv4.